brave-cxf/src/main/java/com/github/kristofa/brave/cxf/BraveServerInInterceptor.java

```diff
@@ -0,0 +1,39 @@
+package com.github.kristofa.brave.cxf;
+
+import static com.github.kristofa.brave.cxf.BraveCxfConstants.BRAVE_SERVER_SPAN;
+
+import org.apache.cxf.interceptor.Fault;
+import org.apache.cxf.interceptor.StaxInInterceptor;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.phase.AbstractPhaseInterceptor;
+import org.apache.cxf.phase.Phase;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.http.HttpServerRequestAdapter;
+import com.github.kristofa.brave.http.SpanNameProvider;
+
+/**
+ * @author Michał Podsiedzik
```

Comment:
please scrub author tags.. unless you want bugs introduced later to be attributed to you :)